src/main/java/com/yandex/money/api/model/showcase/components/container/Paragraph.java

```diff
@@ -13,6 +14,11 @@ private Paragraph(Builder builder) {
         super(builder);
     }
 
+    @Override
+    protected ToStringBuilder getToStringBuilder() {
+        return super.getToStringBuilder().setName("Paragraph");
```

Comment:
It's a little bit confusing. Somewhere you use ```super.getToStringBuilder()....``` to get ```ToStringBuilder```, in other places you use this: ```new ToStringBuilder("Group").append(super.getToStringBuilder())```. The first one is preferable - only one instance is created.